### ISSUE TITLE
Images not appearing

### ISSUE THREAD

**Issue by advil64** [nutrivide/nutrivide.github.io#2]
Yash's picture on the main page and Akshay's and Jo's pictures in the respective modals are not appearing and thus being replaced with the alt text. Possible directory issue, but I've been unable to figure it out.
